docs/User/ProjectRelated/Changelog.md

```diff
@@ -14,11 +14,48 @@
     writeWithBufferOf -> writeWith
     writeChunksWithBufferOf -> writeChunksWith
 
+* Following APIs have been renamed in Streamly.Data.Fold module:
+    head -> one
+
+* Moved Streamly.Data.Array from streamly to streamly-core package.
+* Moved Streamly.Data.Unfold from streamly to streamly-core package.
+* Moved Streamly.Console.Stdio from streamly to streamly-core package.
+* Moved Streamly.FileSystem.Handle from streamly to streamly-core package.
+* Moved Streamly.Unicode.Stream from streamly to streamly-core package.
+
+* Following internal modules moved from streamly to streamly-core:
+    Streamly.Internal.Console.Stdio
+    Streamly.Internal.Data.Array.Stream.Foreign
+    Streamly.Internal.Data.Binary.Decode
+    Streamly.Internal.FileSystem.Handle
+    Streamly.Internal.Unicode.Stream
+    Streamly.Internal.Unicode.String
+    Streamly.Internal.Unicode.Char.Parser
+    Streamly.Internal.Unicode.Array.Char
```

Comment:
We can remove this. Just add a simple note "If you cannot find an internal module, it may have been moved to streamly-core package or may have been renamed.".